code/modules/clothing/neck/_neck.dm

```diff
@@ -185,26 +185,113 @@
+		if(!lungs)//sanity check, enusure patient actually has lungs
+			render_list += "<span class='danger ml-1'>[M] doesn't have any lungs!</span>\n"
+		else
+			if(carbon_patient.stat == DEAD || (HAS_TRAIT(carbon_patient, TRAIT_FAKEDEATH)) || (HAS_TRAIT(carbon_patient, TRAIT_NOBREATH))|| carbon_patient.failed_last_breath || carbon_patient.losebreath)//If pt is dead or otherwise not breathing
+				render_list += "<span class='danger ml-1'>[M.p_theyre(TRUE)] not breathing!</span>\n"
+			else if(lungs.damage > 10)//if breathing, check for lung damage
+				render_list += "<span class='danger ml-1'>You hear fluid in [M.p_their()] lungs!</span>\n"
+			else if(oxy_loss > 10)//if they have suffocation damage
+				render_list += "<span class='danger ml-1'>[M.p_theyre(TRUE)] breathing heavily!</span>\n"
+			else
+				render_list += "<span class='notice ml-1'>[M.p_theyre(TRUE)] breathing normally.</span>\n"//they're okay :D
+
+	//assess heart & blood level
+	if(body_part == BODY_ZONE_CHEST)//if we're listening to the chest
+		if(!heart)//sanity check, ensure the patient actually has a heart
+			render_list += "<span class='danger ml-1'>[M] doesn't have a heart!</span>\n"
+		else
+			if(!heart.beating || carbon_patient.stat == DEAD)
+				render_list += "<span class='danger ml-1'>You don't hear a heartbeat!</span>\n"//they're dead or their heart isn't beating
+			else if(heart.damage > 10 || carbon_patient.blood_volume <= BLOOD_VOLUME_OKAY)
+				render_list += "<span class='danger ml-1'>You hear a weak heartbeat.</span>\n"//their heart is damaged, or they have critical blood
+			else
+				render_list += "<span class='notice ml-1'>You hear a healthy heartbeat.</span>\n"//they're okay :D
+
+	if(body_part != BODY_ZONE_CHEST && body_part != BODY_ZONE_PRECISE_GROIN && body_part != BODY_ZONE_PRECISE_EYES && body_part != BODY_ZONE_PRECISE_MOUTH)//we're checking their pulse on an extremity
+		if(!heart)//sanity check, ensure the patient actually has a heart
+			render_list += "<span class='danger ml-1'>[M] doesn't have a heart!</span>\n"
+		else
+			if(!heart.beating || carbon_patient.blood_volume <= BLOOD_VOLUME_OKAY || carbon_patient.stat == DEAD)
+				render_list += "<span class='danger ml-1'>You can't find a pulse!</span>\n"//they're dead, their heart isn't beating, or they have critical blood
+			else
+				if(heart.damage > 10)
+					heart_strength = span_danger("irregular")//their heart is damaged
+				else
+					heart_strength = span_notice("regular")//they're okay :D
+
+				if(carbon_patient.blood_volume <= BLOOD_VOLUME_SAFE && carbon_patient.blood_volume > BLOOD_VOLUME_OKAY)
+					pulse_pressure = span_danger("thready")//low blood
+				else
+					pulse_pressure = span_notice("strong")//they're okay :D
+
+				render_list += "<span class='notice ml-1'>[M.p_their(TRUE)] pulse is [pulse_pressure] and [heart_strength].</span>\n"
+
+	//assess abdominal organs
+	if(body_part == BODY_ZONE_PRECISE_GROIN)
```

Comment:
if you make it a switch then u move it before your gigacheck on 249 u will be able to make 249 a little more readable

Reply:
It hadn't occurred to me that organizing the code by vital sign involved so much repetition, good catch. It's all grouped by body part in the switch now!